src/coreclr/src/gc/gc.cpp

```diff
@@ -20547,9 +20547,8 @@ size_t gc_heap::get_total_heap_size()
     size_t total_heap_size = 0;
 
 #ifdef MULTIPLE_HEAPS
-    int hn = 0;
 
-    for (hn = 0; hn < gc_heap::n_heaps; hn++)
+    for (int hn = 0; hn < gc_heap::n_heaps; hn++)
```

Comment:
This is a style change. If it pulls it into line with the rest of th efie or project then it's probably fine but if you're going to make style changes can you note them in the PR description please.